test/test_events.py

```diff
@@ -16,6 +16,20 @@ def test_receiving_events():
     eq(event[1], 'py!')
     eq(event[2], [vim.current.buffer.number])
 
+@with_setup(setup=cleanup)
+def test_sending_notify():
+    # notify after notify
+    vim.session.notify('vim_command', "let g:test = 3")
+    cmd = 'call rpcnotify(%d, "test-event", g:test)' % vim.channel_id
```

Comment:
I think you should use `string.format()` instead of `%` op.
`%` is removed in Python3.